models/_sector/dex/trades/ethereum/platforms/_schema.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+
+models:
+  - name: defiswap_ethereum_base_trades
+    meta:
+      blockchain: ethereum
+      sector: dex
+      project: defiswap
+      contributors: pandajackson42, 0xRob, hosuke
+    config:
+      tags: [ 'ethereum', 'dex', 'trades', 'defiswap' ]
+    description: "Defiswap ethereum base trades"
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - evt_index
+            - trace_address
+      - dbt_utils.equal_rowcount:
+          compare_model: ref('defiswap_ethereum_trades')
+      - equal_rowcount_with_sources:
+          evt_sources:
+            - source('defiswap_ethereum', 'CroDefiSwapPair_evt_Swap')
```

Comment:
I add two kinds of rowcount tests (suggested in #3279).
model comparison failed with diff of 3.
https://dune.com/queries/2441243?d=1

Reply:
The diff may be caused by the difference in refreshing time.